tools/packaging/packaging.psm1

```diff
@@ -408,73 +402,28 @@ function New-UnixPackage {
             "libc6",
             "libcurl3",
             "libgcc1",
-            "libssl1.0.0",
+            "libgssapi-krb5-2",
+            "liblttng-ust0",
             "libstdc++6",
-            "libtinfo5",
             "libunwind8",
             "libuuid1",
-            "zlib1g"
+            "zlib1g",
+            "libssl1.0.0",
+            "libicu-dev"
```

Comment:
We should probably open an issue to see if we can get to use the runtime package instead of the dev package.

Reply:
Please add one more task to the existing #3961 to track it.